game/static/game/css/auth.css

```
@@ -10,21 +15,28 @@ body {
     background-color: #0f0f1a;
     color: #d0d0d0;
     font-family: 'Segoe UI', system-ui, -apple-system, sans-serif;
+
     min-height: 100vh;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)